npm/nameSpaceController.go

```diff
@@ -85,17 +76,18 @@ type nameSpaceController struct {
 	nameSpaceLister       corelisters.NamespaceLister
 	nameSpaceListerSynced cache.InformerSynced
```

Comment:
I do not see a place where we are using this has synced variable, can we remove it if not required.

Reply:
Good catch.
NPM checks sync of all controller in npm.go - https://github.com/Azure/azure-container-networking/blob/master/npm/npm.go#L151.  So good to delete this in all controllers.

Reply:
Oh, it is used in each controller UT, but I am not sure it is necessary yet. 
I will check it.  If it does not have issue, I will delete them.